packages/cli/src/commands/blob/command.ts

```diff
@@ -0,0 +1,209 @@
+export const listSubcommand = {
+  name: 'list',
+  aliases: ['ls'],
+  description: 'List all files in the Blob store',
+  arguments: [],
+  options: [
+    {
+      name: 'limit',
+      shorthand: 'l',
+      type: Number,
+      deprecated: false,
+      description:
+        'Number of results to return per page (default: 10, max: 1000)',
+      argument: 'NUMBER',
+    },
+    {
+      name: 'cursor',
+      shorthand: 'c',
+      type: String,
+      deprecated: false,
+      description: 'Cursor from previous page to start listing from',
+      argument: 'STRING',
+    },
+    {
+      name: 'prefix',
+      shorthand: 'p',
+      type: String,
+      deprecated: false,
+      description: 'Prefix to filter Blobs by',
+      argument: 'STRING',
+    },
+    {
+      name: 'mode',
+      shorthand: 'm',
+      type: String,
+      deprecated: false,
+      description: 'Mode to filter Blobs by either folded or expanded',
+      argument: 'String',
+      choices: ['folded', 'expanded'],
+    },
+  ],
+  examples: [],
+} as const;
+
+export const putSubcommand = {
+  name: 'put',
+  aliases: [],
+  description: 'Upload a file to the Blob store',
+  arguments: [
+    {
+      name: 'pathToFile',
+      required: true,
+    },
+  ],
+  options: [
+    {
+      name: 'add-random-suffix',
+      shorthand: 'r',
+      type: Boolean,
+      deprecated: false,
+      description: 'Add a random suffix to the file name',
+      argument: 'Boolean',
+    },
+    {
+      name: 'pathname',
+      shorthand: 'p',
+      type: String,
+      deprecated: false,
+      description: 'Pathname to upload the file to',
+      argument: 'String',
+    },
+    {
+      name: 'multipart',
+      shorthand: 'u',
+      type: Boolean,
+      deprecated: false,
+      description:
+        'If true upload the file in multiple smaller chunks. Default: true',
+      argument: 'Boolean',
+    },
+    {
+      name: 'content-type',
+      shorthand: 't',
+      type: String,
+      deprecated: false,
+      description:
+        'Overwrite the content-type. Will be infered from the file extension if not provided',
```

Comment:
```suggestion
        'Overwrite the content-type. Will be inferred from the file extension if not provided',
```

Also you could perhaps just reuse the docs we have in @vercel/blob